problems/arraysWithLoops.js

```diff
@@ -66,15 +103,25 @@ function findSmallest() {}
  * @returns {number} second smallest value
  */
 
-function findSecondSmallest() {}
+function findSecondSmallest(nums) {
+  let secondSmallest = nums[0]
+  for(let i = 0; i <= nums.length -1; i++)
+  if (nums[i] <= secondSmallest) secondSmallest = nums [i]
```

Comment:
you will need to add an extra condition to make sure that you are keep track of the second smallest number